### PR TITLE
Feat: Getting Started with Solang Guide

### PR DESCRIPTION
**Pull request by ZYJLiu**

### CODE REVIEW

File: developers/guides/solang/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+   [Node.js](https://nodejs.org/en) installed on your system. Windows users will
+   also need to have
+   [Windows Subsystem for Linux](https://solana.com/developers/guides/setup-local-development#windows-users-only)
+   set up.
+2. **Solana Tool Suite Installation**: Begin by installing the
+   [Solana Tool Suite](https://docs.solana.com/cli/install-solana-cli-tools),
+   which includes the Solana Command Line Interface (CLI) and the latest version
+   of Solang.
+3. **Anchor Framework Installation**: Next, install the
+   [Anchor Framework](https://www.anchor-lang.com/docs/installation). Anchor a
```
Comment (nickfrosty) [inline developers/guides/solang/getting-started.md:48]:
```suggestion
   [Anchor Framework](https://www.anchor-lang.com/docs/installation). Anchor is a
```

File: developers/guides/solang/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+   widely used framework in the Solana ecosystem used to simplify the process of
```
Comment (nickfrosty) [inline developers/guides/solang/getting-started.md:49]:
```suggestion
   widely used framework in the Solana ecosystem and simplifies the process of
```

File: developers/guides/solang/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+anchor init project_name --solidity
+```
+
+This command generates a new project with a basic Solang smart contract and a
+test file that demonstrate how to interact with the contract from the client.
+
+## Smart Contract Overview
+
+Next, let’s go over the starter code beginning with the contract itself. Within
+your project's `./solidity` directory, you’ll find the following contract below:
```
Comment (nickfrosty) [inline developers/guides/solang/getting-started.md:81]:
```suggestion
your project's `./solidity` directory, you’ll find the following contract below, which includes:
```

File: developers/guides/solang/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+
+The contract includes:
```
Comment (nickfrosty) [inline developers/guides/solang/getting-started.md:83]:
```suggestion

```

File: developers/guides/solang/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+            value = !value;
+    }
+
+    /// Simply returns the current value of our `bool`.
+    function get() public view returns (bool) {
+            return value;
+    }
+}
+```
+
```
Comment (nickfrosty) [inline developers/guides/solang/getting-started.md:113]:
```suggestion
### Important Differences

```

File: developers/guides/solang/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+   pay the SOL required to create the account for storing the state variable.
+
+```solidity
+@payer(payer) // payer for the "data account"
+constructor(address payer) {
+    print("Hello, World!");
+}
+```
+
+Here's an important distinction between EVM smart contracts and Solana programs:
```
Comment (nickfrosty) [inline developers/guides/solang/getting-started.md:139]:
```suggestion
### Storing of state data

An important distinction between EVM smart contracts and Solana programs is how each stores "state" variables/data:
```

**Review by nickfrosty — CHANGES_REQUESTED** [review]

**Comment by nickfrosty** [comment]
Content looks good, just some minor nits

**Review by ZYJLiu — COMMENTED** [review]
Thanks nick!

**Review by nickfrosty — APPROVED** [review]

**Pull request merged**